src/SerializeAST.cc

```diff
@@ -823,66 +830,88 @@ void findNamedDataStructures(const snowcrash::Elements& elements, DataStructures
 
     }
 }
-#endif
 
-sos::Object drafter::WrapBlueprint(const Blueprint& blueprint, const ASTType astType)
+void registrateNamedMSONStructures(const snowcrash::Elements& elements, refract::Registry& registry) 
 {
-    sos::Object blueprintObject;
-
-#if _WITH_REFRACT_
     DataStructures found;
-    findNamedDataStructures(blueprint.content.elements(), found);
+    findNamedDataStructures(elements, found);
```

Comment:
BTW, this function is named `findNamedDataStructures` since May-29 will we rename it again in next three months?